schema/bom-1.6.proto

```diff
@@ -133,8 +133,8 @@ message Component {
   repeated Component components = 21;
   // Specifies optional, custom, properties
   repeated Property properties = 22;
-  // Specifies optional license and copyright evidence. Only the first item in the optional repeated list is to be taken into account; every other item in the list is to be ignored/omitted.
```

Comment:
> Only the first item in the optional repeated list is to be taken into account; every other item in the list is to be ignored/omitted.

this was an early idea to solve the same issue. it was never publishe / released.
therefore, it is not binding.